test/cql-pytest/test_null.py

```diff
@@ -63,3 +63,22 @@ def test_insert_null_key(cql, table1):
         cql.execute(stmt, [s, None])
     with pytest.raises(InvalidRequest, match='null value'):
         cql.execute(stmt, [None, s])
+
+def test_primary_key_in_null(cql, table1):
+    '''Tests handling of "key_column in ?" where ? is bound to null.'''
+    s = random_string()
+    cql.execute(f"INSERT INTO {table1} (p,c) VALUES ('{s}', '{s}')")
```

Comment:
Nitpick (it is silly enough that I'll commit anyway): This insertion isn't needed. The queries below will fail on input validation, it doesn't matter if a row exists or not. 
Even s = random_string() isn't needed. You can use 'dog' just as well ;-)

Reply:
True.  The insertion is a vestige of an ancient test version that expected a result, before I remembered to check what Cassandra does.

Reply:
Ok, I committed this version already (I can live with the extra millisecond the useless insert adds ;-)), so if you want to fix this, please send a followup patch.